BlackJackets-ui/src/pages/Contact.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+//import "../pages/Contact.css";
+
+function Contact() {
+  const navigate = useNavigate();
+
+  const onSubmit = async (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+
+    formData.append("access_key", "719cf01b-e37a-4b5c-a4db-e5b7b8710a5d");
+
+    const object = Object.fromEntries(formData);
+    const json = JSON.stringify(object);
+
+    const res = await fetch("https://api.web3forms.com/submit", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+      body: json,
+    }).then((res) => res.json());
+
+    if (res.success) {
+      //   console.log("Success", res);
+      alert("Email has sent successfully!");
+      navigate("/guest-view");
```

Comment:
This actually returns them to the login page for some reason. Should it return to the venue page they were on? or the giglisting, or the list of all gigs?